src/io/flutter/logging/FlutterLogEntryParser.java

```diff
@@ -81,13 +80,40 @@ private static FlutterLogEntry parseLoggingEvent(@NotNull Event event) {
       }
     }
 
-    // TODO: If message.getValueAsStringIsTruncated() is true, we'll need to retrieve the full string
+    // TODO(pq): If message.getValueAsStringIsTruncated() is true, we'll need to retrieve the full string
     // value and update this entry after creation.
     String messageStr = message.getValueAsString();
     if (message.getValueAsStringIsTruncated()) {
       messageStr += "...";
     }
-    return new FlutterLogEntry(timestamp(event), category, level, messageStr);
+
+    final FlutterLogEntry entry = new FlutterLogEntry(timestamp(event), category, level, messageStr);
+
+    final Instance data = new Instance(logRecord.getAsJsonObject().get("error").getAsJsonObject());
```

Comment:
consider using a CompletableFuture so you can await the future when showing the summary view so that you don't show null if there is a race condition.